### PR TITLE
format for real

### PR DESCRIPTION
**Pull request by blaggacao**
TL;DR : this is precursor code backed "generally accepted" canonicalization of k8s yaml files to empower file based diffing strategies to consolidate the different sources of truth `./minikube` / `./services`.

---

`kustomize cfg fmt` will probably the canonical way of formatting kubernetes resources in the future.

Additionally, my recommendation is to incorporate this command into pre-commit tooling (if desired). Over `prettier` it does ensure most appropriate semantic order of k8s resources.

From [its documentation](https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/docs/commands/fmt.md):
> Field ordering roughly follows the ordering defined in the source Kubernetes resource definitions (i.e. go structures), falling back on lexicographical sorting for unrecognized fields.

This semantic ordering is of great help for consistency &mdash; I've seen a lot of naive copy+paste & _"oops, forgot this / that"_ during work on #1041 with `services/**/*.yaml`. This is a first step towards mitigation and resolution.

Please look at the individual commits to see with witch commands the results can be reproduced independently.

Note that:
- it catches some collateral `yml` files &mdash; I thought those are no big deal to just subsume under `kustomize cfg fmt`
- if `/minikube` or `/dev-tools` shall be spared, for example, please simply cherry pick the commits your interested in; I'd hope thought to get `./services/` and `./platform` clean and this technique already catches a few schema errors such as in #1053 and similar other previous commits

@weberjm As a relatively massive chore, it would be nice if this could be merged quickly during expected low turnover, e.g. this weekend?

---

After this canonicalization to get from here to `minikube`, one would have to apply the following patches:

```console
$ ls services | xargs -n1 -I{} sh -c 'diff -u services/{}/k8s/deployment.yaml minikube/4-Services/{}/k8s/deployment.yaml >> {}.patch'
$ diff -u services/iam/k8s/deployment.yaml minikube/2-IAM/deployment.yaml >> iam.patch

$ cat $(fd '\.patch') | wl-copy
```
<br/><br/>

<details>
<summary><b>Diff from here to <code>./minikube</code> state</b></summary>

```diff
--- services/app-directory/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/app-directory/k8s/deployment.yaml	2020-09-17 20:28:33.165058779 -0500
@@ -1,10 +1,13 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: app-directory
   namespace: oih-dev-ns
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: app-directory
   template:
     metadata:
       labels:
@@ -19,19 +22,19 @@
         - name: IAM_TOKEN
           valueFrom:
             secretKeyRef:
-              name: app-directory
-              key: iamtoken
+              name: shared-secret
+              key: "iamtoken"
         - name: MONGODB_CONNECTION
           valueFrom:
             secretKeyRef:
-              name: app-directory
-              key: mongourl
+              name: shared-secret
+              key: "appDirectoryMongourl"
         - name: "INTROSPECT_TYPE"
           value: "basic"
         - name: "INTROSPECT_ENDPOINT_BASIC"
-          value: "http://iam.openintegrationhub.com/api/v1/tokens/introspect"
+          value: "http://iam.oih-dev-ns.svc.cluster.local:3099/api/v1/tokens/introspect"
         - name: ORIGIN_WHITELIST
-          value: openintegrationhub.com,http://web-ui.openintegrationhub.com,https://web-ui.openintegrationhub.com
+          value: localoih.com,http://web-ui.localoih.com,https://web-ui.localoih.com
         - name: PORT
           value: "3000"
         - name: API_BASE
--- services/attachment-storage-service/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/attachment-storage-service/k8s/deployment.yaml	2020-09-17 20:28:33.165058779 -0500
@@ -25,21 +25,17 @@
         - name: REDIS_CONFIG
           valueFrom:
             secretKeyRef:
-              name: attachment-storage-service
+              name: shared-secret
               key: REDIS_CONFIG
         - name: PORT
           value: "3002"
         - name: IAM_TOKEN
           valueFrom:
             secretKeyRef:
-              name: attachment-storage-service
-              key: IAM_TOKEN
+              name: shared-secret
+              key: iamtoken
         - name: LOG_LEVEL
           value: trace
-        resources:
-          limits:
-            cpu: 0.1
-            memory: 512Mi
         livenessProbe:
           httpGet:
             port: 3002
--- services/audit-log/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/audit-log/k8s/deployment.yaml	2020-09-17 20:28:33.165058779 -0500
@@ -22,17 +22,17 @@
         - name: MONGODB_URL
           valueFrom:
             secretKeyRef:
-              name: audit-log
-              key: "mongourl"
+              name: shared-secret
+              key: "auditLogMongourl"
         - name: IAM_TOKEN
           valueFrom:
             secretKeyRef:
-              name: audit-log
+              name: shared-secret
               key: "iamtoken"
         - name: "INTROSPECT_TYPE"
           value: "basic"
         - name: "INTROSPECT_ENDPOINT_BASIC"
-          value: "http://iam.openintegrationhub.com/api/v1/tokens/introspect"
+          value: "http://iam.oih-dev-ns.svc.cluster.local:3099/api/v1/tokens/introspect"
         - name: "QUEUE_URL"
           value: "amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local:5672"
         - name: "EXCHANGE_NAME"
@@ -42,7 +42,7 @@
         resources:
           limits:
             cpu: "0.1"
-            memory: 500Mi
+            memory: 1000Mi
         livenessProbe:
           httpGet:
             port: 3007
--- services/component-orchestrator/k8s/deployment.yaml	2020-09-17 20:26:35.211866498 -0500
+++ minikube/4-Services/component-orchestrator/k8s/deployment.yaml	2020-09-17 20:26:39.135901368 -0500
@@ -24,18 +24,20 @@
         image: "openintegrationhub/component-orchestrator:latest"
         env:
         - name: COMPONENT_REPOSITORY_BASE_URL
-          value: "http://component-repository-service.oih-dev-ns.svc.cluster.local:1234"
+          value: "http://component-repository.oih-dev-ns.svc.cluster.local:1234"
         - name: SNAPSHOTS_SERVICE_BASE_URL
           value: "http://snapshots-service.oih-dev-ns.svc.cluster.local:1234"
         - name: SECRET_SERVICE_URL
           value: "http://secret-service.oih-dev-ns.svc.cluster.local:3000/api/v1"
         - name: ATTACHMENT_STORAGE_SERVICE_BASE_URL
           value: "http://attachment-storage-service.oih-dev-ns.svc.cluster.local:3002"
+        - name: "INTROSPECT_ENDPOINT_BASIC"
+          value: "http://iam.oih-dev-ns.svc.cluster.local:3099/api/v1/tokens/introspect"
         - name: IAM_TOKEN
           valueFrom:
             secretKeyRef:
-              name: component-orchestrator
-              key: IAM_TOKEN
+              name: shared-secret
+              key: iamtoken
         - name: LISTEN_PORT
           value: "1234"
         - name: LOG_LEVEL
@@ -43,8 +45,8 @@
         - name: MONGODB_URI
           valueFrom:
             secretKeyRef:
-              name: component-orchestrator
-              key: MONGODB_URI
+              name: shared-secret
+              key: componentOrchestratorMongourl
         - name: RABBITMQ_URI
           value: >-
             amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local
@@ -56,10 +58,6 @@
             http://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local:15672/
         - name: SELF_URL
           value: "http://api-service.oih-dev-ns.svc.cluster.local"
-        resources:
-          limits:
-            cpu: 0.1
-            memory: 512Mi
         livenessProbe:
           httpGet:
             port: 1234
@@ -70,7 +68,7 @@
             path: /healthcheck
         terminationMessagePath: /dev/termination-log
         terminationMessagePolicy: File
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
         securityContext:
           privileged: false
   strategy:
--- services/component-repository/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/component-repository/k8s/deployment.yaml	2020-09-17 20:26:39.139901404 -0500
@@ -25,27 +25,24 @@
         - name: MONGODB_URI
           valueFrom:
             secretKeyRef:
-              name: component-repository
-              key: MONGODB_URI
-        - name: RABBITMQ_URI
-          value: "amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local"
+              name: shared-secret
+              key: componentRepoMongourl
         - name: PORT
           value: "1234"
+        - name: RABBITMQ_URI
+          value: >-
+            amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local
         - name: IAM_TOKEN
           valueFrom:
             secretKeyRef:
-              name: component-repository
-              key: IAM_TOKEN
+              name: shared-secret
+              key: iamtoken
         - name: LOG_LEVEL
           value: trace
         - name: "INTROSPECT_ENDPOINT_BASIC"
-          value: "http://iam.openintegrationhub.com/api/v1/tokens/introspect"
+          value: "http://iam.oih-dev-ns.svc.cluster.local:3099/api/v1/tokens/introspect"
         - name: CORS_ORIGIN_WHITELIST
-          value: "openintegrationhub.com,http://web-ui.openintegrationhub.com"
-        resources:
-          limits:
-            cpu: 0.1
-            memory: 512Mi
+          value: "localoih.com,http://web-ui.localoih.com,http://localhost:3000"
         livenessProbe:
           httpGet:
             port: 1234
--- services/data-hub/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/data-hub/k8s/deployment.yaml	2020-09-17 20:28:33.165058779 -0500
@@ -25,26 +25,17 @@
         - name: MONGODB_URI
           valueFrom:
             secretKeyRef:
-              name: data-hub
-              key: MONGODB_URI
-        - name: RABBITMQ_URI
-          valueFrom:
-            secretKeyRef:
-              name: data-hub
-              key: RABBITMQ_URI
+              name: shared-secret
+              key: dataHubMongourl
         - name: PORT
           value: "1234"
         - name: IAM_TOKEN
           valueFrom:
             secretKeyRef:
-              name: data-hub
-              key: IAM_TOKEN
+              name: shared-secret
+              key: iamtoken
         - name: LOG_LEVEL
           value: trace
-        resources:
-          limits:
-            cpu: 0.1
-            memory: 512Mi
         livenessProbe:
           httpGet:
             port: 1234
--- services/dispatcher-service/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/dispatcher-service/k8s/deployment.yaml	2020-09-17 20:28:33.165058779 -0500
@@ -22,28 +22,28 @@
         - name: MONGODB_URL
           valueFrom:
             secretKeyRef:
-              name: dispatcher-service
-              key: "mongourl"
+              name: shared-secret
+              key: "dispatcherMongoUrl"
         - name: IAM_TOKEN
           valueFrom:
             secretKeyRef:
-              name: dispatcher-service
+              name: shared-secret
               key: "iamtoken"
         - name: "INTROSPECT_TYPE"
           value: "basic"
         - name: "INTROSPECT_ENDPOINT_BASIC"
-          value: "http://iam.openintegrationhub.com/api/v1/tokens/introspect"
+          value: "http://iam.oih-dev-ns.svc.cluster.local:3099/api/v1/tokens/introspect"
         - name: "QUEUE_URL"
           value: "amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local:5672"
         - name: "FLOWREPO_URL"
-          value: "http://flow-repository.openintegrationhub.com"
+          value: "http://flow-repository.oih-dev-ns.svc.cluster.local:3001"
         - name: FLOW_TOKEN
           valueFrom:
             secretKeyRef:
-              name: dispatcher-service
-              key: "flowtoken"
+              name: shared-secret
+              key: "iamtoken"
         - name: "SDF_ID"
-          value: "5d2484d2a422ca001bda5690"
+          value: "ReplaceMe"
         resources:
           limits:
             cpu: "0.1"
--- services/flow-repository/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/flow-repository/k8s/deployment.yaml	2020-09-17 20:26:39.143901440 -0500
@@ -22,27 +22,25 @@
         - name: MONGODB_URL
           valueFrom:
             secretKeyRef:
-              name: flow-repository
-              key: "mongourl"
+              name: shared-secret
+              key: "flowRepoMongourl"
         - name: IAM_TOKEN
           valueFrom:
             secretKeyRef:
-              name: flow-repository
+              name: shared-secret
               key: "iamtoken"
         - name: QUEUE_URL
           value: "amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local:5672"
-        - name: ORIGINWHITELIST
-          value: "openintegrationhub.com,http://web-ui.openintegrationhub.com"
-        - name: NODE_ENV
-          value: "production"
         - name: "INTROSPECT_TYPE"
           value: "basic"
         - name: "INTROSPECT_ENDPOINT_BASIC"
-          value: "http://iam.openintegrationhub.com/api/v1/tokens/introspect"
+          value: "http://iam.oih-dev-ns.svc.cluster.local:3099/api/v1/tokens/introspect"
+        - name: ORIGINWHITELIST
+          value: "localoih.com,http://web-ui.localoih.com,http://localhost:3000"
         resources:
           limits:
             cpu: "0.1"
-            memory: 500Mi
+            memory: 1000Mi
         livenessProbe:
           httpGet:
             port: 3001
--- services/iam/k8s/deployment.yaml	2020-09-17 20:26:35.223866605 -0500
+++ minikube/2-IAM/deployment.yaml	2020-09-17 20:26:39.147901476 -0500
@@ -1,10 +1,13 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: iam
   namespace: oih-dev-ns
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: iam
   template:
     metadata:
       labels:
@@ -16,6 +19,10 @@
         ports:
         - containerPort: 3099
         env:
+        - name: LOG_LEVEL
+          value: error
+        - name: RABBITMQ_URI
+          value: amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local
         - name: IAM_MONGODB_CONNECTION
           valueFrom:
             secretKeyRef:
@@ -26,13 +33,11 @@
         - name: IAM_PORT
           value: "3099"
         - name: IAM_BASEURL
-          value: "https://iam.openintegrationhub.com"
+          value: "http://iam.oih-dev-ns.svc.cluster.local:3099"
         - name: IAM_ORIGINWHITELIST
-          value: "openintegrationhub.com,iam.oih-dev-ns.svc.cluster.local:3099"
+          value: "http://localhost:3000,localoih.com,127.0.0.1,.oih-dev-ns.svc.cluster.local"
         - name: NODE_ENV
           value: "production"
-        - name: RABBITMQ_URI
-          value: "amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local:5672"
         - name: IAM_JWT_SECRET
           valueFrom:
             secretKeyRef:
@@ -68,14 +73,6 @@
             secretKeyRef:
               name: iam
               key: oidc_client_secret
-        resources:
-          limits:
-            cpu: "0.1"
-            memory: 1000Mi
-        volumeMounts:
-        - name: oidc-certs
-          readOnly: true
-          mountPath: /usr/src/app/keystore
         livenessProbe:
           httpGet:
             port: 3099
@@ -88,9 +85,5 @@
             path: /healthcheck
           initialDelaySeconds: 10
           timeoutSeconds: 1
-      volumes:
-      - name: oidc-certs
-        secret:
-          secretName: oidc-certs
   minReadySeconds: 10
   revisionHistoryLimit: 2
--- services/ils/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/ils/k8s/deployment.yaml	2020-09-17 20:28:33.165058779 -0500
@@ -22,17 +22,12 @@
         - name: MONGODB_URL
           valueFrom:
             secretKeyRef:
-              name: ils
-              key: "mongourl"
-        - name: SCHEMA_TOKEN
-          valueFrom:
-            secretKeyRef:
-              name: ils
-              key: "iamtoken"
+              name: shared-secret
+              key: "ilsMongourl"
         resources:
           limits:
             cpu: "0.1"
-            memory: 500Mi
+            memory: 200Mi
         livenessProbe:
           httpGet:
             port: 3003
--- services/meta-data-repository/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/meta-data-repository/k8s/deployment.yaml	2020-09-17 20:26:39.139901404 -0500
@@ -1,10 +1,13 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: meta-data-repository
   namespace: oih-dev-ns
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: meta-data-repository
   template:
     metadata:
       labels:
@@ -19,25 +22,21 @@
         - name: IAM_TOKEN
           valueFrom:
             secretKeyRef:
-              name: meta-data-repository
+              name: shared-secret
               key: iamtoken
         - name: MONGODB_CONNECTION
           valueFrom:
             secretKeyRef:
-              name: meta-data-repository
-              key: mongourl
-        - name: INTROSPECT_TYPE
+              name: shared-secret
+              key: metaDataMongourl
+        - name: "INTROSPECT_TYPE"
           value: "basic"
-        - name: ORIGINWHITELIST
-          value: openintegrationhub.com
         - name: RABBITMQ_URI
           value: amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local:5672
-        - name: INTROSPECT_ENDPOINT_BASIC
-          value: "http://iam.openintegrationhub.com/api/v1/tokens/introspect"
-        - name: BASE_URL
-          value: "http://metadata.openintegrationhub.com"
-        - name: URLS_WITH_PORT
-          value: "false"
+        - name: "INTROSPECT_ENDPOINT_BASIC"
+          value: "http://iam.oih-dev-ns.svc.cluster.local:3099/api/v1/tokens/introspect"
+        - name: ORIGINWHITELIST
+          value: "localoih.com,http://web-ui.localoih.com,http://localhost:3000"
         - name: PORT
           value: "3000"
         - name: API_BASE
--- services/scheduler/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/scheduler/k8s/deployment.yaml	2020-09-17 20:26:39.135901368 -0500
@@ -22,20 +22,18 @@
       - name: scheduler
         image: "openintegrationhub/scheduler:latest"
         env:
+        - name: LOG_LEVEL
+          value: trace
         - name: MONGODB_URI
           valueFrom:
             secretKeyRef:
-              name: scheduler
-              key: MONGODB_URI
+              name: shared-secret
+              key: schedulerMongourl
         - name: RABBITMQ_URI
           value: >-
             amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local
         - name: LISTEN_PORT
           value: "1234"
-        resources:
-          limits:
-            cpu: 0.1
-            memory: 512Mi
         livenessProbe:
           httpGet:
             port: 1234
--- services/secret-service/k8s/deployment.yaml	2020-09-17 20:28:33.169058824 -0500
+++ minikube/4-Services/secret-service/k8s/deployment.yaml	2020-09-17 20:28:33.165058779 -0500
@@ -1,10 +1,13 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: secret-service
   namespace: oih-dev-ns
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: secret-service
   template:
     metadata:
       labels:
@@ -16,24 +19,25 @@
         ports:
         - containerPort: 3000
         env:
-        - name: ORIGINWHITELIST
-          value: openintegrationhub.com
+        - name: CRYPTO_DISABLED
+          value: "false"
+        - name: CRYPTO_KEY
+          valueFrom:
+            secretKeyRef:
+              name: shared-secret
+              key: cryptoKey
         - name: IAM_TOKEN
           valueFrom:
             secretKeyRef:
-              name: secret-service
+              name: shared-secret
               key: iamtoken
         - name: MONGODB_CONNECTION
           valueFrom:
             secretKeyRef:
-              name: secret-service
-              key: mongourl
-        - name: CRYPTO_KEY
-          valueFrom:
-            secretKeyRef:
-              name: secret-service
-              key: crypto_key
-        - # - name: IAM_OIDC_SERVICE_CLIENT_SECRET
+              name: shared-secret
+              key: secretServiceMongourl
+        - #   value: 8ce00058-5f75-435f-9026-ad952480e15a
+          # - name: IAM_OIDC_SERVICE_CLIENT_SECRET
           #   valueFrom:
           #     secretKeyRef:
           #       name: secret-service
@@ -41,12 +45,12 @@
           name: "INTROSPECT_TYPE" # - name: IAM_OIDC_SERVICE_CLIENT_ID
           value: "basic"
         - name: "INTROSPECT_ENDPOINT_BASIC"
-          value: "http://iam.openintegrationhub.com/api/v1/tokens/introspect"
+          value: "http://iam.oih-dev-ns.svc.cluster.local:3099/api/v1/tokens/introspect"
         - name: PORT
           value: "3000"
         - name: API_BASE
           value: "/api/v1"
-        - name: LOG_LEVEL
+        - name: LOGGING_LEVEL
           value: "error"
         - name: TTL_AUTHFLOW
           value: "2m"
@@ -54,6 +58,8 @@
           value: "false"
         - name: RABBITMQ_URI
           value: "amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local:5672"
+        - name: "ORIGINWHITELIST"
+          value: "localoih.com,http://web-ui.localoih.com,http://localhost:3000"
         resources:
           limits:
             cpu: "0.1"
--- services/web-ui/k8s/deployment.yaml	2020-09-17 20:26:35.211866498 -0500
+++ minikube/4-Services/web-ui/k8s/deployment.yaml	2020-09-17 20:26:39.147901476 -0500
@@ -1,10 +1,13 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: web-ui
   namespace: oih-dev-ns
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: web-ui
   template:
     metadata:
       labels:
@@ -18,22 +21,22 @@
         env:
         - name: PORT
           value: "3000"
-        - name: ENDPOINT_SECRETS
-          value: "https://skm.openintegrationhub.com/api/v1"
         - name: ENDPOINT_IAM
-          value: "https://iam.openintegrationhub.com"
+          value: "http://iam.localoih.com"
         - name: ENDPOINT_FLOW
-          value: "https://flow-repository.openintegrationhub.com"
+          value: "http://flow-repository.localoih.com"
+        - name: ENDPOINT_COMPONENT
+          value: "http://component-repository.localoih.com"
+        - name: ENDPOINT_SECRETS
+          value: "http://skm.localoih.com/api/v1"
         - name: ENDPOINT_DISPATCHER
-          value: "https://dispatcher-service.openintegrationhub.com"
+          value: "http://dispatcher-service.localoih.com"
         - name: ENDPOINT_METADATA
-          value: "https://metadata.openintegrationhub.com/api/v1"
+          value: "http://metadata.localoih.com/api/v1"
         - name: ENDPOINT_APP_DIRECTORY
-          value: "https://app-directory.openintegrationhub.com/api/v1"
-        - name: ENDPOINT_COMPONENT
-          value: "https://component-repository.openintegrationhub.com"
+          value: "http://app-directory.localoih.com/api/v1"
         - name: ENDPOINT_WEBHOOKS
-          value: "https://webhooks.openintegrationhub.com"
+          value: "http://webhooks.localoih.com"
         resources:
           limits:
             cpu: "0.1"
--- services/webhooks/k8s/deployment.yaml	2020-09-17 20:26:35.215866534 -0500
+++ minikube/4-Services/webhooks/k8s/deployment.yaml	2020-09-17 20:26:39.143901440 -0500
@@ -26,8 +26,8 @@
         - name: MONGODB_URI
           valueFrom:
             secretKeyRef:
-              name: webhooks
-              key: MONGODB_URI
+              name: shared-secret
+              key: webhooksMongourl
         - name: RABBITMQ_URI
           value: >-
             amqp://guest:guest@rabbitmq-service.oih-dev-ns.svc.cluster.local
@@ -36,11 +36,7 @@
         - name: LOG_LEVEL
           value: trace
         - name: CORS_ORIGIN_WHITELIST
-          value: "https://web-ui.openintegrationhub.com"
-        resources:
-          limits:
-            cpu: 0.1
-            memory: 512Mi
+          value: localoih.com,http://web-ui.localoih.com
         livenessProbe:
           httpGet:
             port: 1234
```

</details><br/><br/>

### CODE REVIEW

**Comment by blaggacao** [comment]
I'll be proposing https://github.com/openintegrationhub/openintegrationhub/pull/1041/commits/f96cccb3b92c7260d7b7d4705f2cd26b5c547e71 directly on top of this re-playing the same command line witchery to guarantee semantically valid outcomes &mdash; which in part depends on proper ordering while double checking manually the `git diffs` between `services` & `minikube` to spot additional "question marks", for example.
